ambry-router/src/test/java/com.github.ambry.router/GetManagerTest.java

```diff
@@ -184,6 +200,33 @@ public void testRangeRequestStitchDifferentSizedBlobs() throws Exception {
     }
   }
 
+  /**
+   * Tests the router range request interface on stitched blob segments.
+   * @throws Exception
+   */
+  @Test
+  public void testRangeRequestStitchDifferentSizedBlobsSegments() throws Exception {
+    if (metadataContentVersion > 2) {
+      //Test grabbing every segment with various offset ranges
+      for (int i = 0; i < NUM_STITCHED_CHUNKS; i++) {
+        testGetSuccessStitch(chunkSize * 6 + 11, new GetBlobOptionsBuilder().blobSegment(i)
```

Comment:
If you're using the same blob size in every test case, set blobSize variable and use.

Reply:
done